js_modules/dagster-ui/packages/ui-core/src/asset-graph/useAssetGraphData.tsx

```diff
@@ -40,6 +40,7 @@ export interface AssetGraphFetchScope {
   // This is used by pages where `hideNodesMatching` is only available asynchronously.
   loading?: boolean;
   useWorker?: boolean;
+  useFavorites?: boolean;
```

Comment:
I could see it being a little confusing to have the `useX` nomenclature for something other than a hook. Perhaps `shouldUseFavorites`? `showFavorites`?